src/shared/acl-util.c

```diff
@@ -208,7 +208,7 @@ int acl_search_groups(const char *path, char ***ret_groups) {
         return ret;
 }
 
-int parse_acl(const char *text, acl_t *acl_access, acl_t *acl_default, bool want_mask) {
+int parse_acl(const char *text, acl_t *acl_access, acl_t *acl_default, bool *acl_cond_exec, bool want_mask) {
```

Comment:
Please add an assertion for `acl_cond_exec`.

Reply:
I'm marking these as resolved since this approach is not desired. We need a better way :(